aws/aws-mcp-cli-commands/src/main/java/software/amazon/smithy/java/aws/mcp/cli/commands/AddAwsServiceBundle.java

```diff
@@ -35,22 +35,24 @@ public class AddAwsServiceBundle extends AbstractAddBundle {
     protected Set<String> blockedApis;
 
     @Option(names = "--read-only-apis",
-            description = "Include read only APIs in the MCP server",
-            defaultValue = "true")
-    protected boolean readOnlyApis;
+            description = "Include read only APIs in the MCP server")
+    protected Boolean readOnlyApis;
 
     @Override
     protected CliBundle getNewToolConfig() {
         var bundleBuilder = AwsServiceBundler.builder()
                 .serviceName(awsServiceName);
-        if (readOnlyApis) {
-            // Include read-only apis unless disabled
-            bundleBuilder.readOnlyOperations();
-        }
         if (allowedApis != null) {
             // User explicitly specified allowed APIs - use only those
             bundleBuilder.exposedOperations(allowedApis);
+        } else if (readOnlyApis == null || readOnlyApis) {
```

Comment:
This else should be removed

Reply:
Just removing the else doesn't work for when allowedApis are provided and readOnlyApis isn't.